2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Configure stats pid to 67979
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Loading settings from /Users/felixwille/.config/wandb/settings
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Loading settings from /Users/felixwille/PycharmProjects/Data_Analysis_Project/cleanrl_repo/cleanrl/wandb/settings
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'cleanrl/local_lc_dqn.py', 'program_abspath': '/Users/felixwille/PycharmProjects/Data_Analysis_Project/cleanrl_repo/cleanrl/local_lc_dqn.py', 'program': '/Users/felixwille/PycharmProjects/Data_Analysis_Project/cleanrl_repo/cleanrl/local_lc_dqn.py'}
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:_log_setup():532] Logging user logs to /Users/felixwille/PycharmProjects/Data_Analysis_Project/cleanrl_repo/cleanrl/wandb/run-20241104_163235-eknrg634/logs/debug.log
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:_log_setup():533] Logging internal logs to /Users/felixwille/PycharmProjects/Data_Analysis_Project/cleanrl_repo/cleanrl/wandb/run-20241104_163235-eknrg634/logs/debug-internal.log
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:init():617] calling init triggers
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {'exp_name': 'local_lc_dqn', 'seed': 1, 'torch_deterministic': True, 'cuda': True, 'track': True, 'wandb_project_name': 'Lipschitz', 'wandb_entity': None, 'capture_video': False, 'save_model': True, 'upload_model': False, 'hf_entity': '', 'env_id': 'MountainCar-v0', 'total_timesteps': 500000, 'learning_rate': 0.001, 'num_envs': 1, 'buffer_size': 100000, 'gamma': 0.99, 'tau': 1.0, 'target_network_frequency': 1000, 'batch_size': 64, 'start_e': 1, 'end_e': 0.01, 'exploration_fraction': 0.3, 'learning_starts': 5000, 'train_frequency': 4, 'log_lipschitz_every': 2000}
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:init():667] starting backend
2024-11-04 16:32:35,696 INFO    MainThread:67979 [wandb_init.py:init():671] sending inform_init request
2024-11-04 16:32:35,697 INFO    MainThread:67979 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-11-04 16:32:35,697 INFO    MainThread:67979 [wandb_init.py:init():684] backend started and connected
2024-11-04 16:32:35,699 INFO    MainThread:67979 [wandb_init.py:init():779] updated telemetry
2024-11-04 16:32:35,713 INFO    MainThread:67979 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-11-04 16:32:36,057 INFO    MainThread:67979 [wandb_init.py:init():863] starting run threads in backend
2024-11-04 16:32:36,131 INFO    MainThread:67979 [wandb_run.py:_console_start():2465] atexit reg
2024-11-04 16:32:36,131 INFO    MainThread:67979 [wandb_run.py:_redirect():2313] redirect: wrap_raw
2024-11-04 16:32:36,131 INFO    MainThread:67979 [wandb_run.py:_redirect():2378] Wrapping output streams.
2024-11-04 16:32:36,131 INFO    MainThread:67979 [wandb_run.py:_redirect():2403] Redirects installed.
2024-11-04 16:32:36,132 INFO    MainThread:67979 [wandb_init.py:init():907] run started, returning control to user process
2024-11-04 16:32:36,133 INFO    MainThread:67979 [wandb_run.py:_tensorboard_callback():1556] tensorboard callback: runs/MountainCar-v0__local_lc_dqn__1__1730734354, True
2024-11-04 16:32:37,633 WARNING MsgRouterThr:67979 [router.py:message_loop():77] message_loop has been closed
